### PR TITLE
[OSDOCS-8328]: Adding hosted control planes to Welcome page

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.14+

Issue: https://issues.redhat.com/browse/OSDOCS-8328

Link to docs preview: https://67823--docspreview.netlify.app/openshift-enterprise/latest/welcome/

QE review:
- [x] QE has approved this change.

Additional information: 

This PR is for the MCE 2.4 release that is scheduled for Nov. 20. Do not merge until then.

The links to the RHACM docs will not work until Nov. 20. 

Note to peer reviewers: I suspect that I'll need to add conditional tags around this text, but I don't have a lot of experience with those, so I appreciate any pointers!

### CODE REVIEW

File: welcome/index.adoc [hunk 1]
```diff
@@ -317,3 +317,18 @@ endif::[]
 endif::openshift-enterprise,openshift-webscale,openshift-origin[]
 
 endif::openshift-rosa[]
+
```
Comment (dfitzmau) [inline welcome/index.adoc:320]: Hi @lahinson .

Will updates also be made to the OCP 4.14. Release notes? I see numerous comments in the `ocp-4-14-release-notes.adoc` file that suggests updates on the following lines in the file:

* 375
* 412
* 1281
* 2683
* 2688
* 
Or search for "Hosted Control Planes" in the event the line numbers have changed.

Comment (lahinson) [reply welcome/index.adoc:320]: Hi @dfitzmau - Yes, my updates to the 4.14 release notes are ongoing in this PR: https://github.com/openshift/openshift-docs/pull/66219

I believe @SNiemann15 will also have updates to the 4.14 release notes related to the upcoming IBM Power and IBM Z docs for hosted control planes.

Comment (lahinson) [reply welcome/index.adoc:320]: Also, it looks like the RHACM GA is slipping a bit, so the docs won't be live tomorrow (Nov. 15). I am waiting to find out what the new GA date will be.

**Comment by zanetworker** [comment]
lgtm

File: welcome/index.adoc [hunk 2]
```diff
@@ -317,3 +317,18 @@ endif::[]
...
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#hosted-control-planes-manage-kubevirt[Managing hosted control plane clusters on OpenShift Virtualization]
+
+* **Technology Preview features**: Hosted control planes remains available as a Technology Preview feature on the Amazon Web Services platform, and is now available as a Technology Preview feature on the {ibmzProductName} platform. For more information, see the following documentation:
+
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#hosting-service-cluster-configure-aws[Configuring the hosting cluster on AWS (Technology Preview)]
+** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-ibmz[Configuring the hosted cluster on x86 bare metal for IBM Z compute nodes (Technology Preview)]
```
Comment (skrthomas) [inline welcome/index.adoc:332]:
```suggestion
** link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#configuring-hosting-service-cluster-ibmz[Configuring the hosted cluster on 64-bit x86 bare metal for IBM Z compute nodes (Technology Preview)]
```

Comment (skrthomas) [reply welcome/index.adoc:332]: Based on the examples here, I think its supposed to be written as I've suggested. https://redhat-documentation.github.io/supplementary-style-guide/#bit-64-x86

**Review by skrthomas — COMMENTED** [review]
Beautiful, just one nit otherwise lgtm!

File: welcome/index.adoc (hunk 2 above)
Comment (lahinson) [reply welcome/index.adoc]: Fixed. Thanks!

**Comment by bburt-rh** [comment]
@lahinson - This is still in draft mode and can't be merged. Can you please update it so that I can merge it? Thanks!

**Pull request merged**